manim/mobject/text/tex_mobject.py

```diff
@@ -175,8 +175,8 @@ def _modify_special_strings(self, tex):
         tex = self._remove_stray_braces(tex)
 
         for context in ["array"]:
-            begin_in = ("\\begin{%s}" % context) in tex
-            end_in = ("\\end{%s}" % context) in tex
+            begin_in = ("\\begin{%s}" % context) in tex  # noqa: UP031
+            end_in = ("\\end{%s}" % context) in tex  # noqa: UP031
```

Comment:
Maybe?

```suggestion
            begin_in = rf"\begin{{{context}}}" in tex
            end_in = rf"\end{{{context}}}" in tex
```

I don't which one's clearer

Reply:
Personally, I feel like the original (although with a raw string) is the most clear. Too many nested `{}` makes it harder to read  for me.

Reply:
Yep I think I agree as well